src/librustdoc/html/static/css/themes/ayu.css

```diff
@@ -531,7 +531,7 @@ kbd {
 	background-color: #314559;
 	border-color: #5c6773;
 	border-bottom-color: #5c6773;
-	box-shadow-color: #c6cbd1;
+	box-shadow: inset 0 -1px 0 #5c6773;
```

Comment:
Note that I intentionally changed it from `#c6cbd1` to `#5c6773` because the old color was the one used in the dark theme and seemed inconsistent with Ayu.